### ISSUE TITLE
Is there any plan to release your pretrained model on ImageNet?

### ISSUE THREAD

**Issue by hermosayhl** [zhaoyuzhi/Reference-Based-Sketch-Image-Colorization-ImageNet#1]
I want to test some gray images with reference images. Thanks

**Comment by zhaoyuzhi** [comment]
Current trained models have very poor results on ImageNet validation set. I am not sure where the problem occurs since the training samples are good.

After solving the problem I will release the models.

**Comment by hermosayhl** [comment]
> Current trained models have very poor results on ImageNet validation set. I am not sure where the problem occurs since the training samples are good.
> 
> After solving the problem I will release the models.

OK! I'm looking forward to your reply.

**Comment by XYFFFFFF** [comment]
So, the model can work well on gray images ? and need not Sketch?

**Comment by caijillx** [comment]
can you release the pretrained model?

**Comment by XYFFFFFF** [comment]
Hello, I have trained in my own dataSet and have very poor results on the Test set. Do u find the answer

**Comment by zhaoyuzhi** [comment]
Hi all,

Thanks for your replies. I can share some of my own experiences here with you.

Currently, I tried the network architecture used in the original paper (CVPR'20) and some other architectures, like the architecture of "Real-Time User-Guided Image Colorization with Learned Deep Priors" and UNet. Unfortunately, I assume the training strategies proposed in the CVPR'20 paper might be not good on large-scale natural images, e.g., there are severe color artifacts in the generated images on the validation set. By the way, the training strategies were proposed for cartoon images in my understanding so there might be some constraints.

Regards,
Zhao

**Comment by mhuwang** [comment]
> 您好，我在自己的数据集中进行了训练，但在测试集上的结果很差。你找到答案了吗

hello，how to test my pretrained model?where should i put my test images and refenence?do you have any code?thanks very much!
